test/basic_test.dart

```diff
@@ -200,7 +200,7 @@ void main() {
           },
           onError: (error) {
             expect(error, isA<PostgrestException>());
-            expect(error.code, '404');
+            expect(error.code, '42P01');
```

Comment:
Also just a simple question, but why did the error code change here?

Reply:
Previously the parsing failed on error [here](https://github.com/supabase-community/postgrest-dart/pull/84/files#diff-9593abc84c6d61f0d7335e023534ce3e829d59cc8b7bf6cf7119761af075bc85R225), because hint was missing. So the exception got generated [here](https://github.com/supabase-community/postgrest-dart/pull/84/files#diff-9593abc84c6d61f0d7335e023534ce3e829d59cc8b7bf6cf7119761af075bc85R236) instead, which doesn't use the body for code generation.

Reply:
@Vinzent03 Sorry, the `here` link took me to the top of the page like the following. Where was it?
<img width="1358" alt="Screen Shot 2022-08-24 at 18 09 29" src="https://user-images.githubusercontent.com/18113850/186378916-53f03c58-bb81-4038-8937-2672037fa059.png">

Reply:
Fixed

Reply:
Amazing! Thanks for the clarification! I'm just noticing the fact that Postgrest uses an interesting error code huh? 😂
https://postgrest.org/en/latest/errors.html